executor/api/rest/client.go

```diff
@@ -163,7 +158,13 @@ func (smc *JSONRestClient) doHttp(ctx context.Context, modelName string, method
 
 	contentType := response.Header.Get("Content-Type")
 
-	return b, contentType, nil
+	err = nil
```

Comment:
err should already be nil if code has reached this point, why this nil re-assignment?

Reply:
Can line 161 be removed?

Reply:
I approved right before your comment so I've added a `/hold` to block the merge. Feel free to remove it once you are happy @glindsell!